### PR TITLE
Add github action to publish docker image and address peer feedback

### LINKED ISSUE #62: Milestone 3 Peer Feedback (Marco)
Hello Group 19, 

Glad to peer review your project it was really cool. 
Below is my feedback based on tag [`0.2.0`](https://github.com/UBC-MDS/bc_election_turnout/tree/v0.2.0)

- **Documentation**: 
  - It is great. File structure is explicit and well organized. Github workflow is also very well maintained. 
      - P.S. How did you guys get rid of the redundant commits and have only PR's show up in your main repo?? That looks super professional
- **Code**: 
  - Code works. Code is very readable as it is very well modularized. 
- **Analysis and reasoning:** 
  - Analysis is thorough and correct. 
- **Communication**: 
  - Yes, report is very well made and easy to follow. It is also interesting to read the reports in different format. For example, pdf and html offer really different experience.
  - One minor potential improvement - in the Results & Discussion section, the third sentence:
      >  As the p-value p<.001 falls below our alpha threshold...

      This could be misleading to think 0.001 as the alpha value.
- **Suggestions**: 
  - The reference section in README.md seems a bit odd. Not an expert in citation formats, but are the two lines starting with 2018a and 2018b in the right format? (If you guys have made sure this is correct just ignore me)
  - In the Usage section of README.md, could include `make clean` to improve reproducibility
  - Overall, the project milestone is very well done and it was hard for me to spot room for improvement. I am especially impressed by the data cleaning process, as well as the abundant amount of EDA done while maintaining high modularity between the scripts. Great job :)

### LINKED ISSUE #68: Add GH Action to Rebuild and Version Docker Container

### PR DESCRIPTION
**Pull request by ChadNeald**

### CODE REVIEW

**Comment by ChadNeald** [comment]
Actually before this merges I didn't run it with the docker container so I will do that first to make sure the files remain consistent.

**Comment by ChadNeald** [comment]
Ok, that should be better now. I still have the old version of the dockerfile that we made together so maybe afterward we can add in the one that Raf made that is more up to date.

File: doc/bc_election_turnout_report.toc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+\contentsline {subsection}{Summary}{1}{section*.2}%
```
Comment (rtaph) [inline doc/bc_election_turnout_report.toc:1]: Is this file autogenerated, or did you add it manually?

File: Dockerfile [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# Author: Chad Neald
+# Date: 2020-12-11
+
 FROM rocker/verse:latest
+
 RUN Rscript -e "install.packages(c('dataMaid', 'here', 'cowplot', 'docopt', 'GGally', 'ggpubr', 'ggthemes', 'janitor'))"
+
 RUN Rscript -e "tinytex::install_tinytex()"
```
Comment (rtaph) [inline Dockerfile]: probably not an issue, but there is no newline at the end of this file.

File: .github/workflows/publish_docker_image.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          - Rebuilt Docker image and published to DockerHub with new tag
+        draft: false
+        prerelease: false
+    - name: Publish to Registry
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: cneald/bc_election # change this to your DockerHub username and repository
+        username: ${{ secrets.DOCKER_USERNAME }} # you need to add your Docker username to this GitHub repo as a secret
+        password: ${{ secrets.DOCKER_PASSWORD }} # you need to add your Docker password to this GitHub repo as a secret
+        tags: "latest,${{ steps.bump.outputs.new_tag }}"
```
Comment (rtaph) [inline .github/workflows/publish_docker_image.yml]: probably not an issue, but there is no newline at the end of this file.

**Review by rtaph — APPROVED** [review]
Looks good to me!

**Review by kmoravej — APPROVED** [review]

File: doc/bc_election_turnout_report.toc (hunk 1 above)
Comment (ChadNeald) [reply doc/bc_election_turnout_report.toc:1]: It looks like it was auto generated.

**Pull request merged**